ios/engine/KMEI/KeymanEngine/Classes/KeymanPackage.swift

```diff
@@ -300,7 +317,7 @@ public class KeymanPackage {
   }
   
   public func infoHtml() -> String {
-    if let readmeURL = self.readmePageURL {
+    if let readmeURL = self.pageURL(for: .readme) {
       if let html = try? String(contentsOfFile: readmeURL.path, encoding: String.Encoding.utf8) {
```

Comment:
Will this handle an upper-cased filename `Readme.htm`? (e.g. rac_brahui keyboard)

afaict, there's no more Welcome.htm files in the released/ keyboards.

Reply:
iOS is case-sensitive, so... no, it will not.  If we feel it important, I could add in a 'legacy' "Readme.htm" check, I suppose.

Reply:
I think the keyboard package needs to be fixed in this case.